README.md

```diff
@@ -1082,18 +1082,24 @@ Correct number matching must result in 3.4 == 3.400 == 0.34e1 and correct number
 If the search term is a number then a numeric mode is used for "==", "<>", "<", "<=", ">", and ">=" if the actual JSONB type of the stored value is `number`
 (JSONB has no `integer` type).
 
-### CQL2PgJSON: Cross index searches
+### CQL2PgJSON: Cross table index queries
 
-Limited cross table searches are supported.  If you desire a join across tables the following conditions must be met:
+CQL2PgJSON supports cross table joins via subquery.  This allows arbitrary depth relationships in parent->child and child->parent relationships.  Foreign keys must be uni-directional(parent->child and child->parent).
```

Comment:
Foreign keys must be uni-directional, the direction is always from the foreign key field in the child table to the primary key id field of the parent table.

Reply:
I am not sure what should be changed here so I removed the line.